app/server.go

```diff
@@ -1094,6 +1095,24 @@ func runLicenseExpirationCheckJob(a *App) {
 	}, time.Hour*24)
 }
 
+func runReportToAWSMeterJob(a *App) {
+	model.CreateRecurringTask("Collect and send usage report to AWS Metering Service", func() {
+		doReportUsageToAWSMeteringService(a)
+	}, time.Hour*1)
+}
+
+func doReportUsageToAWSMeteringService(a *App) {
+	awsMeter := awsmeter.New(a.srv.Store, a.Config())
+	if awsMeter == nil {
+		mlog.Error("Cannot obtain instance of AWS Metering Service.")
+		return
+	}
+
+	dimensions := []string{model.AWS_METERING_DIMENSION_USAGE_HRS}
+	reports := awsMeter.GetUserCategoryUsage(dimensions, time.Now().UTC(), time.Now().Add(-1*time.Hour).UTC())
+	awsMeter.ReportUserCategoryUsage(reports)
+}
```

Comment:
```suggestion
func runReportToAWSMeterJob(s *Server) {
	model.CreateRecurringTask("Collect and send usage report to AWS Metering Service", func() {
		doReportUsageToAWSMeteringService(s)
	}, time.Hour*1)
}

func doReportUsageToAWSMeteringService(s *Server) {
	awsMeter := awsmeter.New(s.Store, s.Config())
	if awsMeter == nil {
		mlog.Error("Cannot obtain instance of AWS Metering Service.")
		return
	}

	dimensions := []string{model.AWS_METERING_DIMENSION_USAGE_HRS}
	reports := awsMeter.GetUserCategoryUsage(dimensions, time.Now().UTC(), time.Now().Add(-1*time.Hour).UTC())
	awsMeter.ReportUserCategoryUsage(reports)
}
```

Reply:
This also need to move the `runReportToAWSMeterJob` call from `app/app.go` file to the `app/server.go` file.

Reply:
sure, same comment as above.